### PR TITLE
Cleanup, fix typos

### PR DESCRIPTION
**Pull request by ok300**

### CODE REVIEW

**Review by dangeross — COMMENTED** [review]
RN codegen needs running,  otherwise looks good

File: lib/core/src/swapper/boltz_status_stream.rs [hunk 1]
```diff
@@ -140,7 +140,9 @@ impl SwapperStatusStream for BoltzStatusStream {
                                                 Err(e) => warn!("WS response is invalid SwapUpdate: {e:?}"),
                                             }
                                         },
-                                        Ok(msg) => info!("Unhandled msg: {msg:?}"),
+                                        Ok(Message::Ping(_)) => debug!("Received ping"),
```
Comment (dangeross) [inline lib/core/src/swapper/boltz_status_stream.rs:143]: The nice thing to do here would be to send back a Pong

Comment (ok300) [reply lib/core/src/swapper/boltz_status_stream.rs:143]: Yep, tungstenite does that automatically, here it only surfaces the received ping event.

Comment (dangeross) [reply lib/core/src/swapper/boltz_status_stream.rs:143]: Cool 👍

**Review by dangeross — APPROVED** [review]

**Pull request merged**